### PR TITLE
Fix two voices fine-tune + add extra voice samples

### PR DESCRIPTION
**Pull request by DrewThomasson**
added the the missing slash

### CODE REVIEW

**Comment by DrewThomasson** [comment]
Had to remove the SubZero fine-tune, 

Not good enough, became overFit after training on 8 epoches

**Comment by DrewThomasson** [comment]
We can remove any that have too much noise or you think needs modifications done to the audio

**Comment by ROBERT-MCDOWELL** [comment]
more the audio is clear,  with presence and no noise, better it is indeed.
you can use the normalize script.

**Comment by DrewThomasson** [comment]
Does E2A not already normalize it?

**Comment by ROBERT-MCDOWELL** [comment]
> Does E2A not already normalize it?

are you talking about the fine tuning or the 16khz 24khz voice you create in ./voices ?
